app/views/wikies/index.html.erb

```diff
@@ -80,7 +80,7 @@
 
 <% @allowed_collaborations.each do |wiky| %>
       <tr>
-        <td><%= wiky.title %></td>
+        <td><%= wiky.reload.title %></td>
```

Comment:
<td><%= wiky.reload.title %></td>

I removed reload. It was unnecessary.

On Sun, Feb 15, 2015 at 6:08 AM, Eliot Sykes <notifications@github.com>
wrote:

> In app/views/wikies/index.html.erb
> <https://github.com/LuisMenjivar/wiki/pull/57#discussion_r24721520>:
>
> > @@ -80,7 +80,7 @@
> >
> >  <% @allowed_collaborations.each do |wiky| %>
> >        <tr>
> > -        <td><%= wiky.title %></td>
> > +        <td><%= wiky.reload.title %></td>
>
> @LuisMenjivar <https://github.com/LuisMenjivar> Try this without the
> .reload call, so it is just wiky.title. Is the reload call needed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LuisMenjivar/wiki/pull/57/files#r24721520>.
>